engine/uState.ml

```diff
@@ -663,7 +663,7 @@ let subst_univs_context_with_def def usubst (ctx, cst) =
   (LSet.diff ctx def, UnivSubst.subst_univs_constraints usubst cst)
 
 let is_trivial_leq (l,d,r) =
```

Comment:
I think this function should be named in a more informative way. In particular changing the code and not the name and not seeing it as a bugfix is very fishy.